docker/test/integration/features/http.feature

```diff
@@ -18,10 +19,28 @@ Feature: Sending data using InvokeHTTP to a receiver using ListenHTTP
     And the "success" relationship of the ListenHTTP processor is connected to the PutFile
 
     When both instances start up
-    Then a flowfile with the content "test" is placed in the monitored directory in less than 30 seconds
+    Then at least one flowfile with the content "test" is placed in the monitored directory in less than 120 seconds
+
+  # Failing scenario, needs to be fixed
+  @skip
+  Scenario: Multiple files transfered via HTTP are received and transferred only once
+    Given a GetFile processor with the "Input Directory" property set to "/tmp/input"
+    And the "Keep Source File" of the GetFile processor is set to "false"
+    And two files with content "first message" and "second message" are placed in "/tmp/input"
+    And a InvokeHTTP processor with the "Remote URL" property set to "http://secondary:8080/contentListener"
+    And the "HTTP Method" of the InvokeHTTP processor is set to "POST"
+    And the "success" relationship of the GetFile processor is connected to the InvokeHTTP
+
+    And a ListenHTTP processor with the "Listening Port" property set to "8080" in a "secondary" flow
+    And a PutFile processor with the "Directory" property set to "/tmp/output" in the "secondary" flow
+    And the "success" relationship of the ListenHTTP processor is connected to the PutFile
+
+    When both instances start up
+    Then two flowfiles with the contents "first message" and "second message" are placed in the monitored directory in less than 120 seconds
```

Comment:
can this scenario be added (when fixed) in a later PR?  it is not related to ConsumeKafka

Reply:
Added Jira ticket https://issues.apache.org/jira/browse/MINIFICPP-1588